src/store/modules/webhooks/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Module } from 'vuex'
+import WebhooksState from './WebhooksState'
+import RootState from '@/store/RootState'
+import getters from './getters'
+import mutations from './mutations'
+import actions from './actions'
+
+const webhooksModule: Module<WebhooksState, RootState> = {
+  namespaced: true,
+  state: {
+    cached: {
```

Comment:
I think we can define the cached state as empty.

Reply:
updated